### DISCUSSION TITLE
自定义UA支持

### DISCUSSION THREAD

**Discussion by LetMeDecay** [kkkgo/PaoPaoGateWay#102] [想法]
有些机场会根据ua下发不同的配置，比如非meta核心的端会自动过滤掉hy2节点
而订阅转换又会透传ua，跟直连机场节点订阅没区别
手动配置成clash-verge的版本就能获取到hy2节点了

**Comment by kkkgo** [comment]
你说的是suburl的ua吗，默认是“ClashforWindows/clash-verge/Clash/clash"”

**Reply by LetMeDecay** [reply]
对的，我用的机场订阅有点毛病，有时候更新订阅会出现 去除x条不合适线路，但是在win下clash-verge更新订阅就没问题

**Reply by kkkgo** [reply]
是这个项目吗 https://github.com/clash-verge-rev/clash-verge-rev

**Reply by LetMeDecay** [reply]
是的，使用这个软件自带的ua就能更新出hy2节点 `clash-verge/v1.6.4`

**Reply by kkkgo** [reply]
最新版本已优化。https://github.com/kkkgo/PaoPaoGateWay/releases
